### ISSUE TITLE
FLUX image changes a lot with -0.2 resharpen

### ISSUE THREAD

**Issue by tazztone** [Haoming02/sd-webui-resharpen#12]
amazing extension as i always found FLUX to be oversharpened by default.
however is it normal that the image changes so much?
LEFT side resharpen ON -0.2.                                  RIGHT is OFF
![2024-08-21 15 52 16](https://github.com/user-attachments/assets/ecc4e71f-1deb-4549-bf8b-8a25a7794668)
both use the same seed and prompt.
`Steps: 14, Sampler: Euler, Schedule type: Simple, Model: fluxFusionDS_v0_Q8, , Resharpen Sharpness: -0.2, Resharpen Scaling: Flat`
cheers

**Comment by Haoming02** [comment]
Interesting

I still haven't experimented with Flux yet. And since it uses a completely different VAE that now has ***16*** channels, I might need to change how the Extension works for Flux specifically.

**Comment by tazztone** [comment]
already happy with the results tbh. just big changes with the same seed. cheers

**Comment by tazztone** [comment]
i will hijack my issue for the error i just encountered:

resharpen is unchecked yet it still makes my generation fail (with Flux **nf4** model):
```
  File "C:\_stability_matrix\Data\Packages\stable-diffusion-webui-forge\extensions\sd-webui-resharpen\scripts\resharpen.py", line 33, in hijack_callback
    delta: torch.Tensor = d["x"].detach().clone() - params.cache
: The size of tensor a (96) must match the size of tensor b (256) at non-singleton dimension 3
```
disabling the extension was the only fix.

**Comment by Haoming02** [comment]
Forgot to fix the exact same issue I had on the ComfyUI implementation XD

**Comment by tazztone** [comment]
>  I might need to change how the Extension works for Flux specifically.

do you have any plans for flux support in general? 
will you eventually combine your useful extensions into one that can do-it-all?

**Comment by Haoming02** [comment]
> do you have any plans for flux support in general?

Yeah, just didn't have much time to experiment with Flux. ~~And my office PC is so slow to run Flux...~~

**Comment by Haoming02** [comment]
Welp, after [playing around with Flux](https://github.com/Haoming02/sd-webui-latent-playground), I cannot find anything special aside from the fact that **Flux** feels more sensitive to these changes compared to **SD1** and **SDXL**. On top of that, these changes become stronger the fewer your `Steps` is. So it might not be suitable for `schnell` models. 

**TL;DR:** Keep the effect strength smaller and use higher *(`20`+)* steps
